src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
```

Comment:
i try to stick with the code style that is used in the projects i added and i saw there is https://github.com/robertkowalski/couchdb-setup/blob/2594-2598-number-of-nodes/src/setup.erl#L21